### ISSUE TITLE
update link for userscript.org

### ISSUE THREAD

**Issue by ghost** [os0x/NinjaKit#9]
link to userscript.org no longer functional. Please update to http://userscripts-mirror.org. Thank you

**Comment by meat-ball** [comment]
I don't believe many, if any are posting updates to that site. It hasn't been managed for quite a awhile, and is filled with malware. You can find ninjakit on one of the current JS sites.
Also os0x hasn't touched this in years from what it looks like. May want to consider grabbing a fork.
